nautobot/core/apps/__init__.py

```diff
@@ -124,6 +124,11 @@ def register_homepage_panels(path, label, homepage_layout):
     define different parts of the layout.
 
     These objects are converted into a dictionary to be stored inside of the Nautobot registry.
+
+    Args:
+        path (str): Path to the app which defines the homepage layout.
+        label (str): Label of the app which defines the homepage layout.
```

Comment:
`path` and `label` are **only** relevant when using custom templates, correct? Otherwise they're unused?

Reply:
Correct, i'll add a coment